### PR TITLE
WIP: Adds Instructions for Installing on FreeBSD

### PR DESCRIPTION
**Pull request by tuaris**
This adds installation instructions for FreeBSD using the port or package.  I'm not familiar with Hugo so I apologize if something is incorrect.

NOTE: Don't merge until the port has been added to FreeBSD.  I will remove the "WIP" prefix once that's done.

### CODE REVIEW

File: content/en/docs/Installation/freebsd.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+---
+title: "FreeBSD Install"
+linkTitle: "FreeBSD"
+date: 2020-03-14
```
Comment (deluan) [inline content/en/docs/Installation/freebsd.md:4]: Docsy sorts the menu by this date. Please change it so "FreeBSD" shows up after "macOS". You can see the results of your PR here: https://deploy-preview-32--navidrome.netlify.app/docs/installation/

**Review by deluan — CHANGES_REQUESTED** [review]
Thanks for creating the port and for documenting it!

I don't have means to test it, and I'm assuming you are going to maintain the port, right?

File: content/en/docs/Installation/freebsd.md (hunk 1 above)
Comment (tuaris) [reply content/en/docs/Installation/freebsd.md:4]: Interesting...  If I am understanding this correctly, in order to get things arranged correctly, I need to change the macOS date to 2017-01-04 and set FreeBSD's to 2017-01-03?

**Comment by deluan** [comment]
Hey @tuaris, has the port been added to FreeBSD already? Can we merge this?

**Comment by tuaris** [comment]
@deluan this can be merged, Navidrome has been added to the official FreeBSD ports tree: https://www.freshports.org/multimedia/navidrome/

**Pull request merged**